client/my-sites/site-settings/composing/after-the-deadline.jsx

```diff
@@ -0,0 +1,198 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+import { connect } from 'react-redux';
+
+/**
+ * Internal dependencies
+ */
+import JetpackModuleToggle from '../jetpack-module-toggle';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLegend from 'components/forms/form-legend';
+import FormSettingExplanation from 'components/forms/form-setting-explanation';
+import TokenField from 'components/token-field';
+import FormToggle from 'components/forms/form-toggle';
+import InfoPopover from 'components/info-popover';
+import ExternalLink from 'components/external-link';
+import { getSelectedSiteId } from 'state/ui/selectors';
+import { isJetpackModuleActive } from 'state/selectors';
+
+class AfterTheDeadline extends Component {
+	static defaultProps = {
+		isSavingSettings: false,
+		isRequestingSettings: true,
+		fields: {}
+	};
+
+	static propTypes = {
+		handleToggle: PropTypes.func.isRequired,
+		setFieldValue: PropTypes.func.isRequired,
+		isSavingSettings: PropTypes.bool,
+		isRequestingSettings: PropTypes.bool,
+		fields: PropTypes.object,
+	};
+
+	onChangeIgnoredPhrases = ( phrases ) => {
+		this.props.setFieldValue( 'ignored_phrases', phrases.join( ',' ) );
+	};
+
+	renderToggle( name, isDisabled, label ) {
+		const {
+			fields,
+			handleToggle,
+			isRequestingSettings,
+			isSavingSettings
+		} = this.props;
+		return (
+			<FormToggle
+				className="composing__module-settings-toggle is-compact"
+				checked={ !! fields[ name ] }
+				disabled={ isRequestingSettings || isSavingSettings || isDisabled }
+				onChange={ handleToggle( name ) }
+			>
+				{ label }
+			</FormToggle>
+		);
+	}
+
+	renderProofreadingSection() {
+		const { afterTheDeadlineModuleActive, translate } = this.props;
+
+		return (
+			<div className="composing__module-settings is-indented">
+				<FormLegend>
+					{ translate( 'Proofreading' ) }
+				</FormLegend>
+				<FormSettingExplanation>
+					{ translate( 'Automatically proofread content when:' ) }
+				</FormSettingExplanation>
+
+				{
+					this.renderToggle( 'onpublish', ! afterTheDeadlineModuleActive, translate(
+						'Posts or pages are first published'
```

Comment:
Hi! I've found a possible matching string that has already been translated **45** times:
`translate( 'a post or page is first published' )` *ES Score: 9.02*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).